src/ripple/app/main/Main.cpp

```diff
@@ -161,6 +160,9 @@ void printHelp (const po::options_description& desc)
            "     submit_multisigned <tx_json>\n"
            "     tx <id>\n"
            "     validation_create [<seed>|<pass_phrase>|<key>]\n"
+           "     validators"
+           "     validator_list_sites"
+           "     version\n"
            "     wallet_propose [<passphrase>]\n";
 }
```

Comment:
this looks good, but seems like it still doesn't completely match the list at https://github.com/ripple/rippled/blob/5214b3c1b09749420ed0682a2e49fbbd759741e5/src/ripple/rpc/impl/Handler.cpp#L58-L116  -- should it or do we deliberately omit some ?

Reply:
We deliberately omit some of these.

Reply:
Which ones do we deliberately omit? Should we make a notation in `handlerArray` as such?

Reply:
Generally, the ones that require multiple parameters or involve complicated parsing.